src/main/java/bio/terra/service/job/JobBuilder.java

```diff
@@ -33,11 +37,10 @@ public JobBuilder addParameter(String keyName, Object val) {
 
         // check that keyName doesn't match one of the required parameter names
         // i.e. disallow overwriting one of the required parameters
-        boolean isParameterRequired = keyName.equals(JobMapKeys.DESCRIPTION.getKeyName())
-            || keyName.equals(JobMapKeys.FLIGHT_CLASS.getKeyName())
```

Comment:
Why do we no longer store the flight class in the job parameter map?

Reply:
FLIGHT_CLASS was never used anywhere except here.